cmd/dptp-controller-manager/main.go

```diff
@@ -267,6 +264,10 @@ func main() {
 		logrus.Infof("Loaded %q context from in-cluster config", appCIContextName)
 	}
 
+	if _, hasRegistryCluster := kubeconfigs[opts.registryClusterName]; !hasRegistryCluster {
```

Comment:
We will `Fatalf`  in line 262  if that kubeconfig doesn't exist so I don't think this check adds any value

Reply:
nvm, I misread.
/lgtm